lib/sonatype-nexus3-stack.ts

```diff
@@ -14,15 +14,15 @@ export class SonatypeNexus3Stack extends cdk.Stack {
 
     // deploy sonatype-nexus3 chart
     const domainName = this.node.tryGetContext('domainName');
-    if (!domainName)
-      throw new Error('Must specify the custom domain name.');
-
     const stack = cdk.Stack.of(this);
 
     var hostedZone = null;
     var certificate: certmgr.Certificate | undefined;
     const r53Domain = this.node.tryGetContext('r53Domain');
     if (r53Domain) {
+      if (!domainName)
```

Comment:
it would be nice to add a test case only if `r53Domain` is specified.

Reply:
Test case included in latest commit.